Detectors/MUON/MID/Base/src/GeometryTransformer.cxx

```diff
@@ -0,0 +1,113 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// \file   MID/Base/src/GeometryTransformer.cxx
+/// \brief  Geometry transformer for MID
+/// \author Diego Stocco <Diego.Stocco at cern.ch>
+/// \date   07 July 2017
+
+#include "MIDBase/GeometryTransformer.h"
+
+#include <fstream>
+#include <string>
+#include "Math/GenVector/RotationX.h"
+#include "Math/GenVector/RotationY.h"
+#include "FairLogger.h"
+#include "MIDBase/Constants.h"
+
+namespace o2
+{
+namespace mid
+{
+
+//______________________________________________________________________________
+GeometryTransformer::GeometryTransformer()
+{
+  /// Default constructor
+  init();
+}
+
+//______________________________________________________________________________
+void GeometryTransformer::init()
+{
+  /// Initializes default geometry
+  const double degToRad = 3.14159265358979312 / 180.;
```

Comment:
I believe such constants are already defined in ROOT somewhere. Maybe we can avoid redefining here?